custom-fonts.md

```diff
@@ -1,12 +1,30 @@
 ---
 layout: page
-title: Using Custom Fonts
+title: Using Fonts
 
 permalink: /custom-fonts/
 ---
 
-You can use custom fonts in your Flutter application by including them in your
-`pubspec.yaml` file under the `fonts` heading in your `flutter` section:
+## Introduction
+
+You can use different fonts in your Flutter app.
+For example, you may have a custom font that your designer created,
+or you may have a font from [Google Fonts](https://fonts.google.com/).
+
+This page describes how to configure fonts for your Flutter app,
+and use them when rendering text.
+
+
+## Using fonts
+
+Using fonts in your Flutter application is done in two steps. 
+First include them in your `pubspec.yaml`, 
+second declare the use in [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html) property.
+
+### First declare in the font asset
+
+First delcare your font like in the `pubspec.yaml` example below.
+And be sure to copy the font to the directory listed in the `pubspec.yaml` instructions.
```

Comment:
We don't usually start a sentence with "And be". Perhaps "Then, copy the font file to the location specified in your pubspec.yaml".